docs/_writing_multiple_outputs.rst

```diff
@@ -0,0 +1,248 @@
+Multiple Output Files
+===========================================
+
+Tools which create more than one output file are common.  There are several different methods to accommodate this need.  Each one of these has their advantages and weaknesses; careful thought should be employed to determine the best method for a particular tool.
+
+-------------------------------------------
+Static Multiple Outputs
+-------------------------------------------
+
+Handling cases when tools create a static number of outputs is simple.  Simply include an ``<output>`` tag for each output desired within the tool XML file:
+
+.. code-block:: xml
+
+    <tool id="example_tool" name="Multiple output" description="example">
+        <command>example_tool.sh $input1 $tool_option1 $output1 $output2</command>
+        <inputs>
+            ...
+        </inputs>
+        ...
+        <outputs>
+            <data format="interval" name="output1" metadata_source="input1" />
+            <data format="pdf" name="output2" />
+        </outputs>
+    </tool>
+  
+---------------------------------------------------------
+Static Outputs Determinable from Inputs 
+---------------------------------------------------------
+
+In cases when the number of output files varies, but can be determined based upon a user's parameter selection, the use of the filter tag can be used.  The text contents of the ``<filter>`` tag are ``eval`` ed and if the expression is ``True`` a dataset will be created as normal.  If the expression is ``False``, the output dataset will not be created; instead a ``NoneDataset`` object will be created and made available. When used on the command line the text ``None`` will appear instead of a file path. The local namespace of the filter has been populated with the tool parameters.
```

Comment:
parameter selection, the use of the filter tag can be used. -> arameter selection, the filter tag can be used.